cli/src/cli_util.rs

```diff
@@ -771,7 +771,11 @@ impl WorkspaceCommandHelper {
     ) -> Result<IndexSet<Commit>, CommandError> {
         let mut all_commits = IndexSet::new();
         for revision_str in revision_args {
-            let (expression, all) = self.parse_revset_with_all_prefix(revision_str)?;
+            let (expression, modifier) = self.parse_revset_with_modifier(revision_str)?;
+            let all = match modifier {
+                Some(RevsetModifier::All) => true,
+                None => false,
+            };
             if all {
```

Comment:
Nit: I think `if let Some(RevsetModifier::All) = modifier` would be better since there's only that single modifier for now

Reply:
I use `match` for exhaustiveness. It's unknown what kind of `prefix:` will be added.

If we prefer simpler code, maybe we can use `struct RevsetModifierAll` in place of the `enum`.

Reply:
Makes sense, how do you feel about moving the if/else branches into the corresponding match cases? I just find mapping it to a bool and then going through those same cases again a bit redundant, but that's just my preference.

Reply:
I originally coded that way, but reverted because I honestly don't know what will be added to this `match` statement. If the added modifier is invalid in this context, early return would be better. If it's another valid resolution strategy, inlining `if`/`else` should be better.